sound/soc/sof/intel/hda-stream.c

```diff
@@ -433,15 +433,27 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 irqreturn_t hda_dsp_stream_interrupt(int irq, void *context)
 {
 	struct hdac_bus *bus = context;
+	struct sof_intel_hda_dev *sof_hda = bus_to_sof_hda(bus);
+	u32 stream_mask;
 	u32 status;
 
+	/*
+	 * For verbose debugging only, enable it when we doubt
+	 * the IRQ is not generated.
+	 */
```

Comment:
The explanation isn't very useful...
"Intentionally add verbose log before pm_runtime and spin lock"

Reply:
I don't want to add comments with "hardcodeing" consequent invoking, which means we need maintain/update it once those invoking changed, maybe remove this verbose debugging can make everybody happy.